transforms/code/repo_level_ordering/kfp_ray/repo_level_order_wf.py

```diff
@@ -122,23 +122,30 @@ def repo_level_order(
     # Ray cluster
     ray_name: str = "repo_level_order-kfp-ray",
     ray_head_options: dict = {"cpu": 1, "memory": 4, "image": task_image},
-    ray_worker_options: dict = {"replicas": 2, "max_replicas": 2, "min_replicas": 2, "cpu": 2, "memory": 4, "image": task_image},
+    ray_worker_options: dict = {
+        "replicas": 2,
+        "max_replicas": 2,
+        "min_replicas": 2,
+        "cpu": 2,
+        "memory": 4,
+        "image": task_image,
+    },
     server_url: str = "http://kuberay-apiserver-service.kuberay.svc.cluster.local:8888",
     # data access
     data_s3_config: str = "{'input_folder': 'test/repo_level_ordering/input', 'output_folder': 'test/repo_level_ordering/output'}",
     data_s3_access_secret: str = "s3-secret",
     data_max_files: int = -1,
     data_num_samples: int = -1,
     # orchestrator
-    runtime_actor_options: dict = {'num_cpus': 0.8},
+    runtime_actor_options: dict = {"num_cpus": 0.8},
     runtime_pipeline_id: str = "pipeline_id",
-    runtime_code_location: dict = {'github': 'github', 'commit_hash': '12345', 'path': 'path'},
+    runtime_code_location: dict = {"github": "github", "commit_hash": "12345", "path": "path"},
     # repo_level_order parameters
     repo_lvl_stage_one_only: bool = False,
     repo_lvl_grouping_column: str = "repo_name",
     repo_lvl_store_type: str = "ray",
     repo_lvl_store_backend_dir: str = "",
-    repo_lvl_store_ray_cpus: float = "0.5",
+    repo_lvl_store_ray_cpus: float = 0.5,
```

Comment:
Most of the other changes here were pre-commit cleanups, but this one seems like a real bug fix.

Reply:
yes, it was and it is fixed in #619